### ISSUE TITLE
Getting Error: 'ENOTDIR: not a directory, scandir' using UI Tooling version 3.0

### ISSUE THREAD

**Issue by sammok83** [SAP/ui5-tooling#784]
Hi there,

When using **ui5 serve -o** or **fiori run --open**  to run my app, I'm hit with the following error:

_Error: ENOTDIR: not a directory, scandir 'C:\Users\Sam\Desktop\Development\...\initFlpSandbox.js'_

## Expected Behavior
App is able to run successfully without any errors.
FYI, using @ui5/cli": "^2.14.6, it works.

## Current Behavior
App is unable to run successfully with the error:
_Error: ENOTDIR: not a directory, scandir 'C:\Users\Sam\Desktop\Development\...\initFlpSandbox.js'_
This is with "@ui5/cli": "^3.0.0"

## Steps to Reproduce the Issue
Run **ui5 serve -o test/flpSandbox.html#TestApp-manage** on a generated template with any yeoman ui5-generator

## Context
 * UI5 Module Version (output of `ui5 --version` when using the CLI): **`3.0.0`**
 * Node.js Version: **`v18.12.1`**
 * npm Version: `9.4.2`
 * OS/Platform: `Windows 10`
 * Browser *(if relevant)*: `Chrome`
 * Other information regarding your environment *(optional)*: `nothing`

## Log Output / Stack Trace
I've checked the file **initFlpSandbox.js** file and it's there in the path specified

When running ui5 serve with verbose on:
![e2](https://user-images.githubusercontent.com/10509241/219205832-b3b291d3-ce54-4a39-9dd2-0b084880f788.jpg)
 out there.

In Chrome, I'm getting this error
![e1](https://user-images.githubusercontent.com/10509241/219205007-ba760fef-232b-441a-a19a-377f8495aca9.jpg)

**Comment by AramSol** [comment]
we also have the same issue
![image](https://user-images.githubusercontent.com/39582300/219325612-ec5550c0-ac81-4aa3-9b05-a3445f26a36b.png)
in our case we are unable to load ui5 resources

**Comment by RandomByte** [comment]
Thanks for reporting. Could you please also share the content of the relevant `ui5.yaml` file?

We suspect this might be related to https://github.com/SAP/ui5-tooling/issues/779, which will be fixed in the next patch release.

**Comment by flovogt** [comment]
@AramSol Thanks for providing your config. Its indeed a duplicate of [#779](https://github.com/SAP/ui5-tooling/issues/779). Will be fixed in the next patch release.

**Comment by flovogt** [comment]
Fixed with 1cd94f7f15ed07283e198238edb546517ee25691

**Comment by flovogt** [comment]
@AramSol @sammok83 could you please verify if this issue is gone with release `3.0.1` of `@ui5/cli`

**Comment by AramSol** [comment]
@flovog,the `ui5 serve` now work as expected for me but `ui5 build` ignore the exclude resources. all exclude files get also minified and the custom task `ui5-task-zipper` also zipp all the files that listed in excludes section
![image](https://user-images.githubusercontent.com/39582300/219471527-e83cbca5-c0d5-4d29-bae2-01e8501baf6b.png)

**Comment by sammok83** [comment]
Thanks @flovogt. The latest release resolves the issue.

**Comment by RandomByte** [comment]
> @Flovog,the `ui5 serve` now work as expected for me, but now `ui5 build` ignore the exclude resources, all exclude files get also minified and the custom task `ui5-task-zipper` also zip all the files that are listed in excludes section as well. !

Thank you for the quick feedback @AramSol! Indeed, while fixing this issue, we introduced a regression causing the builder excludes to be essentially ignored. This was an unfortunate oversight and we have to investigate how this slipped through our integration test.

I prepared another fix with https://github.com/SAP/ui5-project/pull/582, which we will release ASAP.

CC: @flovogt
